package.json

```diff
@@ -55,20 +52,20 @@
     "typescript": "^3.8.2"
   },
   "dependencies": {
-    "@hapi/hapi": "18.4.1",
-    "@hapi/inert": "5.2.2",
-    "@hapi/joi": "15.1.1",
-    "@hapi/vision": "5.5.2",
+    "@hapi/hapi": "20.2.1",
+    "@hapi/inert": "6.0.5",
+    "@hapi/vision": "6.1.0",
     "convict": "4.3.1",
     "dotenv": "6.0.0",
     "fast-json-stringify": "2.0.0",
-    "hapi-pino": "8.0.1",
-    "hapi-require-https": "3.0.0",
-    "hapi-swaggered": "3.1.0",
-    "hapi-swaggered-ui": "3.1.0",
+    "hapi-pino": "9.1.2",
+    "hapi-require-https": "5.0.0",
+    "hapi-swagger": "^14.2.4",
+    "joi": "^17.6.0",
     "knex": "0.19.5",
-    "mongoose": "5.9.5",
+    "mongoose": "6.1.8",
     "mysql": "2.15.0",
+    "pino-pretty": "^7.5.1",
```

Comment:
Could you make update to this file and change the package versions to exact. In fact you may have to fix your '.npmrc' to automatically install exact versions for the future.

Reply:
Done